scripts/build/build.sh

```diff
@@ -30,7 +30,10 @@ pushd "$MAVEN_DIR"
 echo "-------------------------------------------------------------------------------"
 echo "Building Project"
 echo "-------------------------------------------------------------------------------"
-echo "Running Maven in quiet mode, this may take several minutes."
+echo "Running Maven in quiet mode."
+echo "This build can take anywhere from between 5 mins to an hour."
+echo "Normally, most of this extra time in the first build is taken up compiling Rust. This is then quicker in "
```

Comment:
The "this extra time" and "this is then quicker" seem to refer to some missing explanation of the first build that should be before both of them. There is some explanation of what the first build is doing, but that's sandwiched between these two references, which makes it difficult to parse.

There were estimates here before of the difference in time between the first build and subsequent builds, which we could do with keeping.